taichi/program/kernel.cpp

```diff
@@ -323,9 +323,11 @@ void Kernel::account_for_offloaded(OffloadedStmt *stmt) {
   if (task_type == OffloadedStmt::TaskType::listgen) {
     stat.add("launched_tasks_list_op", 1.0);
     stat.add("launched_tasks_list_gen", 1.0);
-  } else if (task_type == OffloadedStmt::TaskType::clear_list) {
-    stat.add("launched_tasks_list_op", 1.0);
-    stat.add("launched_tasks_list_clear", 1.0);
+  } else if (task_type == OffloadedStmt::TaskType::serial) {
+    // TODO: Do we need to distinguish serial tasks that contain clear lists vs
+    // those who don't?
```

Comment:
Good point :-) For simplicity let's don't do that.

Reply:
Ack, let me keep these TODOs so that we don't forget them if we revisit this part...